Calculate/Calculator.cs

```diff
@@ -0,0 +1,57 @@
+﻿namespace Calculate;
+
+public class Calculator
+{
+    public IReadOnlyDictionary<char, Func<int, int, double>> MathematicalOperations { get; }
+    public Calculator()
+    {
+        MathematicalOperations = new Dictionary<char, Func<int, int, double>>()
+        {
+            {'/', Divide},
+            {'*', Multiply},
+            {'-', Subtract},
+            {'+', Add}
+        };
+    }
+
+    public static double Add(int paramOne, int paramTwo)
+    {
+        return paramOne + paramTwo;
+    }
+
+    public static double Subtract(int paramOne, int paramTwo)
+    {
+        return paramOne - paramTwo;
+    }
+
+    public static double Multiply(int paramOne, int paramTwo)
+    {
+        return paramOne * paramTwo;
+    }
+
+    public static double Divide(int paramOne, int paramTwo)
+    {
+        return (double) paramOne / paramTwo;
+    }
+
+    public static bool TryCalculate(string calculation, out double result)
+    {
+        bool TF = false;
```

Comment:
```suggestion
        bool isValid = false;
```
Typically, you should name your bools with what they are expected to check. Some valid names would be `isValid` `calculationIsSuccessful` `isCalculated` etc.